fix test that fails with PyYAML==5.1b5

Deploy failed due to newer pyyaml installed in circleci: https://circleci.com/gh/Duke-GCB/bespin-cli/116

Error:
```
FAIL: test_write_config (bespin.test_config.ConfigFileTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/circleci/project/bespin/test_config.py", line 53, in test_write_config
    self.assertEqual('{token: Secret, url: someurl}', write_strs.strip())
AssertionError: '{token: Secret, url: someurl}' != 'token: Secret\nurl: someurl'
```
Test is kind of fragile and expects the content to look like JSON(which is allowable in YAML).
https://github.com/Duke-GCB/bespin-cli/blob/f3b1871011f414e1769e6ec7914aabf484fabcf8/bespin/test_config.py#L53

--- a/bespin/test_config.py
+++ b/bespin/test_config.py
@@ -2,7 +2,7 @@ from __future__ import absolute_import
 from unittest import TestCase
 from bespin.config import ConfigFile, Config, ConfigSetupAbandoned, DEFAULT_BESPIN_URL
 from mock import patch, Mock, mock_open
-
+import yaml
 
 class ConfigFileTestCase(TestCase):
     @patch('bespin.config.os')
@@ -50,7 +50,7 @@ class ConfigFileTestCase(TestCase):
             config_file.write_config(config)
         write_call_args = mocked_open.return_value.write.call_args_list
         write_strs = ''.join([acall[0][0] for acall in write_call_args])
-        self.assertEqual('{token: Secret, url: someurl}', write_strs.strip())
+        self.assertEqual({"token": "Secret", "url": "someurl"}, yaml.safe_load(write_strs))
 
     def test_prompt_user_for_token(self):
         config_file = ConfigFile(filename='/tmp/test.yml')

